examples/common/python/connectors/direct/tcs_listener/tcs_work_order_handler.py

```diff
@@ -140,6 +149,37 @@ def WorkOrderSubmit(self, **params):
 
         wo_id = params["workOrderId"]
         input_json_str = params["raw"]
+        input_value_json = json.loads(input_json_str)
+        jrpc_id = input_value_json["id"]
```

Comment:
LGTM flags this as never used:

`jrpc_id = input_value_json["id"]`
--
  **| NEW The value assigned to local variable 'jrpc_id' is never used.**

Reply:
Addressed it.